src/plone/restapi/services/content/tus.py

```diff
@@ -117,9 +117,12 @@ def reply(self):
         tus_upload = TUSUpload(uuid4().hex, metadata=metadata)
 
         self.request.response.setStatus(201)
+        url = self.request.getURL()
+        if url.endswith("@tus-replace"):
+            url = url[:-7] + "upload"
```

Comment:
@instification I am wondering if we shouldn't be more explicit here to make the code easier to read.

Reply:
@tisto I hav updated the code, hopefully it makes it more readable now?